src/views/Predictions/components/Menu.tsx

```diff
@@ -20,22 +20,19 @@ const HelpButtonWrapper = styled.div`
   order: 1;
   margin: 0 2px 0 8px;
 
-  ${({ theme }) => theme.mediaQueries.lg} {
+  ${({ theme }) => theme.mediaQueries.sm} {
     order: 2;
     margin: 0 0 0 8px;
   }
 `
 
 const TimerLabelWrapper = styled.div`
   order: 3;
-  max-width: 100px;
+  width: 100px;
 
   ${({ theme }) => theme.mediaQueries.sm} {
```

Comment:
SM width is enough to put timer label to left most place, therefore we can set width to auto for different languages especially for closing text translation